### PR TITLE
fixed

### LINKED ISSUE #9511: Lots of csp errors are thrown (and reported) when loading add-on detail or developer profile pages
### Describe the problem and steps to reproduce it:
0. Open developer tools network tab
1. Load the detail page for a developer, or an add-on that has other add-ons made by the same developer (don't navigate to it from other pages: copy the URL to your browser location bar and press enter)

### What happened?
- Lots of requests to `__cspreport__` are made

### What did you expect to happen?
- No requests to `__cspreport__` are made.

### Anything else we should know?
- There are a lot of `Content Security Policy: The page’s settings blocked the loading of a resource at inline (“style-src”)` messages in the console tab
- A similar issue was found and fixed in the past: https://github.com/mozilla/addons-frontend/issues/3655

### LINKED ISSUE #9551: The bottom of the permissions box (when fully expanded) does not have rounded corners
Before expanding:

Screenshot 2020-07-29 14 02 11

After expanding:

Screenshot 2020-07-29 14 02 44

### PR DESCRIPTION
**Pull request by Kushagraw12**
Fixes #9551 

I have add bottom radius to these cards. A constant radius value was used.
Looks fine now.
Screenshot after adding the border at the bottom edges:
 Screenshot (67)

### CODE REVIEW

File: src/ui/components/ShowMoreCard/styles.scss [hunk 1]
```diff
@@ -56,7 +56,9 @@
 
 .PermissionsCard {
   .Card-contents {
-    border-radius: 0;
+    // FIX FOR ISSUE 9551
```
Comment (eviljeff) [inline src/ui/components/ShowMoreCard/styles.scss:59]: we don't need a code comment unless it's a particularly obscure fix (and this isn't)

Comment (Kushagraw12) [reply src/ui/components/ShowMoreCard/styles.scss]: Okay Sir, understood!
I have removed the comment.

**Comment by caitmuenster** [comment]
Thanks so much for the patch, @Kushagraw12! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating a profile on addons.mozilla.org? I'd be happy to vouch for you!

**Comment by Kushagraw12** [comment]
> Thanks so much for the patch, @Kushagraw12! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 
> 
> Would you be interested in creating a profile on addons.mozilla.org? I'd be happy to vouch for you! 

Thank you for adding my name to recognition wiki! 🥳
Yeah sure! I would be very much interested in making a profile! ✌🏻

**Pull request merged**